### PR TITLE
Proposal wording for Examples regarding legal exception

### PR DESCRIPTION
**Pull request by gautierchomel**
Also for review: @chrisONIX

### CODE REVIEW

File: UX-Guide-Metadata/draft/principles/index.html [hunk 1]
```diff
@@ -801,24 +801,17 @@ <h3>Legal considerations</h3>
...
-<p>This title may not yet be fully accessible to all readers. Check the list of accessibility features to determine  if this title is suitable for your way of reading.</p>
-</aside>
-<aside class="example" title="Compact explanation">
-<p>May not  be fully accessible. Check features for suitability.</p>
-</aside>
--->
+	<section id="legal-examples">
+		<h2>Examples</h2>
+		<aside class="example" title="Descriptive explanation">
+			<p>This title benefit from an exemption or exception to legal provision of accessible publication.</p>
```
Comment (clapierre) [inline UX-Guide-Metadata/draft/principles/index.html:808]: This sounds awkward, not sure what you are trying to say here. "There are no legal exceptions provided." wouldn't we just not display anything?

**Review by gregoriopellegrino — APPROVED** [review]

File: UX-Guide-Metadata/draft/principles/index.html (hunk 1 above)
Comment (gautierchomel) [reply UX-Guide-Metadata/draft/principles/index.html:808]: Do you have a better wording? 
The intent is to say that the title benefits from an exemption.

Comment (gautierchomel) [reply UX-Guide-Metadata/draft/principles/index.html:808]: Maybe "This title is exempt from the legal provisions for accessible publication."

Comment (gregoriopellegrino) [reply UX-Guide-Metadata/draft/principles/index.html:808]: I suggest to discuss this in the issue :) https://github.com/w3c/publ-a11y/issues/350

**Comment by GeorgeKerscher** [comment]
This is closed because 451 resolves this item

**Pull request closed without merging**